src/EditorFeatures/Core/Tagging/AbstractAsynchronousTaggerProvider.TagSource_ProduceTags.cs

```diff
@@ -208,11 +207,9 @@ private async Task RecomputeTagsAsync(bool highPriority, CancellationToken cance
                         _dataSource.ThreadingContext, _subjectBuffer, DelayTimeSpan.NonFocus, cancellationToken).ConfigureAwait(true);
                 }
```

Comment:
can you merge the other PR in?

Reply:
What other PR?

Reply:
the one that changed this to be NoThrow here.

Reply:
➡️ Rebased the change on main